### PR TITLE
Update the repo after the release of 2.6.1 packages

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

File: nuget/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.Package.csproj [hunk 1]
```diff
@@ -14,15 +14,13 @@
   
   <ItemGroup>
     <AnalyzerNupkgDependency Include="Microsoft.CodeQuality.Analyzers" />
-    <AnalyzerNupkgDependency Include="Microsoft.CodeQuality.Analyzers.Exp" />
```
Comment (mavasani) [inline nuget/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.Package.csproj]: This is a port from [2.6.1](https://github.com/dotnet/roslyn-analyzers/tree/2.6.1) branch that was used to generate the [latest release](https://github.com/dotnet/roslyn-analyzers/releases/tag/v2.6.1)

**Comment by heejaechang** [comment]
@mavasani just curious. what is this about?

**Comment by mavasani** [comment]
@heejaechang We released a [2.6.1 release](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers/2.6.1) of analyzer packages, so we need to update the documentation and infrastructure to move everything forward.

**Comment by mavasani** [comment]
@dotnet-bot retest prtest/Windows/NT/Debug please

**Comment by mavasani** [comment]
Ping for reviews.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #### Analyzer package
 
-Example: Microsoft.CodeQuality.Analyzers, version [2.6.0](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers/2.6.0)
+Example: Microsoft.CodeQuality.Analyzers, version [2.6.1](https://www.nuget.org/packages/Microsoft.CodeQuality.Analyzers/2.6.1)
```
Comment (sharwell) [inline .github/ISSUE_TEMPLATE.md]: :memo: This should not include a default value pointing to a valid target. It's difficult for me to go from a diagnostic ID to a package, so a valid value here makes it impossible to distinguish "I don't know" from "I meant this".

File: README.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ For instructions on using this tutorial, see [Instructions](src/MetaCompilation.
 Microsoft.CodeAnalysis.Analyzers
 --------------------------------
 
-*Latest stable version:* [2.6.0](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers/)
+*Latest stable version:* [2.6.1](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers/)
```
Comment (sharwell) [inline README.md]: :bulb:

```
*Latest stable version:* [![NuGet](https://img.shields.io/nuget/v/Microsoft.CodeAnalysis.Analyzers.svg)](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers)
```

Renders like this:

> *Latest stable version:* [NuGet](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers)

Can be updated for the items below as well, and removes a bunch of the cases where the file needs to be manually edited after releases.

Comment (mavasani) [reply README.md]: Thanks, addressed.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (mavasani) [reply .github/ISSUE_TEMPLATE.md]: Thanks, addressed.

Comment (mavasani) [reply .github/ISSUE_TEMPLATE.md]: Revised it further to not require specifying the analyzer name, but just the diagnostic ID.

File: README.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ Provides common code quality guidelines.
 Microsoft.CodeQuality.Analyzers.Exp
 --------------------------------
 
-*Latest version:* Not Yet Available
+*Latest pre-release version:* [![NuGet](https://img.shields.io/myget/v/Microsoft.CodeQuality.Analyzers.Exp.svg)](https://dotnet.myget.org/feed/roslyn-analyzers/package/nuget/Microsoft.CodeQuality.Analyzers.Exp)
```
Comment (sharwell) [inline README.md]: One of the servers is running slowly right now so I can't verify, but I believe the image here should be this:

```
![MyGet](https://img.shields.io/dotnet.myget/roslyn-analyzers/v/Microsoft.CodeQuality.Analyzers.Exp.svg)
```

Comment (mavasani) [reply README.md]: That package will be deleted from master soon. DFA rules will merge into existing packages in features/dataflow branch.

**Pull request merged**